main/Application/Tasks/GUI/Private/guiHelper.cpp

```
@@ -0,0 +1,336 @@
+        .skip_unhandled_events = false,
+    };
+
+    ESP_ERROR_CHECK(esp_timer_create(&LVGL_TickTimer_args, &p_GUITask_State->LVGL_TickTimer));
+    ESP_ERROR_CHECK(esp_timer_start_periodic(p_GUITask_State->LVGL_TickTimer, CONFIG_GUI_LVGL_TICK_PERIOD_MS * 1000));
+
+    /* Create LVGL clock update timer. Use a 100 ms interval for smoother updates */
+    p_GUITask_State->UpdateTimer[0] = lv_timer_create(GUI_Helper_Timer_ClockUpdate, 100, NULL);
+
+    p_GUITask_State->UpdateTimer[1] = lv_timer_create(GUI_Helper_Timer_SpotUpdate, 2000, NULL);
+    p_GUITask_State->UpdateTimer[2] = lv_timer_create(GUI_Helper_Timer_SpotmeterUpdate, 5000, NULL);
+    p_GUITask_State->UpdateTimer[3] = lv_timer_create(GUI_Helper_Timer_SceneStatisticsUpdate, 5000, NULL);
```

Cppcheck (2.19.0)

[error] Array 'p_GUITask_State->UpdateTimer[3]' accessed at index 3, which is out of bounds.

(arrayIndexOutOfBounds)